code/style.css

```diff
@@ -26,6 +50,9 @@ p {
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  margin-top:20px;
+  margin-bottom: 20px;
```

Comment:
Tip! For margins and padding you can write everything in one line instead of multiple. For this example `margin: 20px 0px;` would have worked. The first stands for bottom and top, and the last stands for right and left